agents/fsevent.go

```diff
@@ -344,6 +345,21 @@ func (fsev FSEvent) AsCDR(timezone string) *engine.CDR {
 	return storCdr
 }
 
+// AsCGREvent converts FSEvent into CGREvent
+func (fsev FSEvent) AsCGREvent(timezone string) (cgrEv *utils.CGREvent, err error) {
+	sTime, err := fsev.GetSetupTime(utils.META_DEFAULT, timezone)
+	if err != nil {
+		return nil, err
+	}
+	cgrEv = &utils.CGREvent{
+		Tenant: fsev.GetTenant(utils.META_DEFAULT),
+		ID:     utils.UUIDSha1Prefix(),
+		Time:   &sTime,
+		Event:  fsev.AsMapStringInterface(timezone),
+	}
+	return cgrEv, nil
```

Comment:
here you just need return since you have loaded cgrEv before and it is already part of your return signature